services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -535,6 +535,12 @@ module.exports = {
       const requiresRegistrationOptions = getAsObject(content, 'mutations.Website.requiresRegistrationOptions');
       const requiresAccessLevels = get(content, 'mutations.Website.requiresAccessLevels');
       const { startDate, endDate } = requiresRegistrationOptions;
+      if (requiresRegistration && (startDate || endDate)) {
```

Comment:
I don't believe you want to change this resolver... if you do, the management interface will no longer function

Reply:
If you're trying to determine if registration is required for gating, that should be handled elsewhere